def divide(first, second):
    #res = first / second  выдает ошибку, перенес в условие
    if second != 0:
        res = first / second
        return res
    else:
        return 'Ошибка'
#result1 = divide(69, 3)

#result2 = divide(3, 0)

    #result3 = true_divide(49, 7)

    #result4 = true_divide(15, 0)

#print(result1)   проверка

#print(result2)

    #print(result3)

    #print(result4)